composer.json

```diff
@@ -6,8 +6,9 @@
     "require": {
         "php": "^8.1",
         "ext-gd": "*",
-        "ext-json": "*",
         "ext-intl": "*",
+        "ext-json": "*",
+        "ext-imagick": "*",
```

Comment:
Any specific reason you prefer imagick over gd? I'm not opposed using imagick but it adds more dependencies to the project and other peoples hosting environment.

```
sudo apt-get install php8.1-imagick
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  fonts-droid-fallback fonts-noto-mono fonts-urw-base35 ghostscript gsfonts imagemagick-6-common libaom3
  libavahi-client3 libavahi-common-data libavahi-common3 libcups2 libdav1d5 libde265-0 libfftw3-double3 libgomp1
  libgs9 libgs9-common libheif1 libidn12 libijs-0.35 libjbig2dec0 liblcms2-2 liblqr-1-0 libltdl7 libmagickcore-6.q16-6
  libmagickwand-6.q16-6 libopenjp2-7 libpaper-utils libpaper1 libwebpdemux2 libwebpmux3 libx265-199 poppler-data
Suggested packages:
  fonts-noto fonts-freefont-otf | fonts-freefont-ttf fonts-texgyre ghostscript-x cups-common libfftw3-bin libfftw3-dev
  liblcms2-utils libmagickcore-6.q16-6-extra poppler-utils fonts-japanese-mincho | fonts-ipafont-mincho
  fonts-japanese-gothic | fonts-ipafont-gothic fonts-arphic-ukai fonts-arphic-uming fonts-nanum
Recommended packages:
  ttf-dejavu-core
The following NEW packages will be installed:
  fonts-droid-fallback fonts-noto-mono fonts-urw-base35 ghostscript gsfonts imagemagick-6-common libaom3
  libavahi-client3 libavahi-common-data libavahi-common3 libcups2 libdav1d5 libde265-0 libfftw3-double3 libgomp1
  libgs9 libgs9-common libheif1 libidn12 libijs-0.35 libjbig2dec0 liblcms2-2 liblqr-1-0 libltdl7 libmagickcore-6.q16-6
  libmagickwand-6.q16-6 libopenjp2-7 libpaper-utils libpaper1 libwebpdemux2 libwebpmux3 libx265-199 php8.1-imagick
  poppler-data
0 upgraded, 34 newly installed, 0 to remove and 0 not upgraded.
Need to get 294 kB/27.6 MB of archives.
After this operation, 106 MB of additional disk space will be used.
```

Reply:
I failed to find a way with the GD library. I think it is a good idea to stick to Imagick which works well - even though it's a dependency